Source/Meadow.Foundation.Peripherals/Sensors.Distance.A02yyuw/Driver/Readme.md

```diff
@@ -2,19 +2,31 @@
 
 **A02yyuw serial ultrasonic distance sensor**
 
-The **A02yyuw** library is included in the **Meadow.Foundation.Sensors.Distance.A02yyuw** nuget package and is designed for the [Wilderness Labs](www.wildernesslabs.co) Meadow .NET IoT platform.
+### Pinout for Sensor:
```

Comment:
These readme files are auto-generated but I'll grab this text and add it to our docs - thx!